include/boost/geometry/algorithms/convex_hull.hpp

```diff
@@ -123,7 +125,7 @@ struct convex_hull<Box, box_tag>
         boost::array<typename point_type<Box>::type, 4> range;
         geometry::detail::assign_box_corners_oriented<Reverse>(box, range);
         geometry::append(out, range);
-        if (Close)
+        if ( BOOST_GEOMETRY_CONDITION(Close) )
```

Comment:
Please remove the extra spacing you added - it is not according to our guidelines, and never used in this sourcefile (at least not in the original version).